sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/indexes/CreateIndexExample.java

```diff
@@ -18,23 +19,65 @@
      */
     private static final String ENDPOINT = Configuration.getGlobalConfiguration().get("AZURE_COGNITIVE_SEARCH_ENDPOINT");
     private static final String ADMIN_KEY = Configuration.getGlobalConfiguration().get("AZURE_COGNITIVE_SEARCH_API_KEY");
-    private static final String INDEX_NAME = "good-food";
 
     public static void main(String[] args) {
-        AzureKeyCredential searchApiKeyCredential = new AzureKeyCredential(ADMIN_KEY);
-
+        // Create the SearchIndex client.
         SearchIndexClient client = new SearchIndexClientBuilder()
             .endpoint(ENDPOINT)
-            .credential(searchApiKeyCredential)
+            .credential(new AzureKeyCredential(ADMIN_KEY))
             .buildClient();
 
-        SearchIndex newIndex = new SearchIndex(INDEX_NAME,
-            Arrays.asList(new SearchField("Name", SearchFieldDataType.STRING).setKey(Boolean.TRUE),
-                new SearchField("Cuisine", SearchFieldDataType.STRING)));
+        // Configure the index using SearchFields
+        String indexName = "hotels";
```

Comment:
From food industry to hospitality? Why the change of heart ;)

Reply:
Turns out being a chef was too stressful and fast paced for me 😄